python/tools/build-wheels.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e -u -x
+
+function repair_wheel {
+    wheel="$1"
+    if ! auditwheel show "$wheel"; then
+        echo "Skipping non-platform wheel $wheel"
+    else
+        auditwheel repair "$wheel" --plat "manylinux2010_x86_64" -w /io/python/wheels/
```

Comment:
Are we producing manylinux1 wheels as well?

Reply:
Yeah `auditwheel` sees that we can and automatically produces them: https://test.pypi.org/project/nvtx/0.1.dev1/#files